docs/ops/memory/mem_setup_jm.md

```diff
@@ -62,7 +62,7 @@ affect the size of the respective components:
 
 As mentioned before in the [total memory description](mem_setup.html#configure-total-memory), another way to set up the memory
 for the Flink Master is to specify explicitly the *JVM Heap* size ([`jobmanager.memory.heap.size`](../config.html#jobmanager-memory-heap-size)).
-It gives more control over the available *JVM Heap* for the following purposes:
+It gives more control over the available *JVM Heap* which contains:
```

Comment:
```suggestion
It gives more control over the available *JVM Heap* which is used by:
```
or `which you can use for`
or other lines have to be formulated as some memory components

Reply:
True, I will correct it.